driver/normalizer/transforms.go

```diff
@@ -3,9 +3,10 @@ package normalizer
 import "gopkg.in/bblfsh/sdk.v2/driver"
 
 var Transforms = driver.Transforms{
-	Namespace:   "java",
-	Preprocess:  Preprocess,
-	Normalize:   Normalize,
-	Annotations: Native,
-	Code:        Code,
+	Namespace:      "java",
```

Comment:
This actually looks like a partial SDK update.

Reply:
Yes, I noticed that as well - how would one verify that?

Reply:
Does this one produce the same result?
```
go run ./vendor/gopkg.in/bblfsh/sdk.v2/cmd/bblfsh-sdk update
```

Reply:
That's what I was thinking about trying after re-reading https://doc.bblf.sh/writing-a-driver/babelfish-sdk.html#updating-sdk-version

```
$ go run vendor/gopkg.in/bblfsh/sdk.v2/cmd/bblfsh-sdk/main.go update --dry-run
$ echo $?
0

$ bblfsh-sdk update --dry-run
managed file changed "driver/normalizer/transforms.go"
changes are required
```

I guess `bblfsh-sdk update` needs to be done together with SDK version update, which would be in a next PR.

What do you think, @dennwc ?

A bit misleading thing was that I got this from a pre-commit hook that calls a newer version of sdk binary and not though the intentional call to it.

Reply:
Yes, this is exactly the problem mentioned in the issue. Currently, you need to `go install` the SDK tool from the `vendor` for it to be in sync with the driver.

Reply:
Got it.

So, for now:
 - I'm just dropping 9d9d4d7
 - to make sure it's not lost and that I did not misunderstand something - would you be so kind to please link in the issue that you mention above?

Reply:
https://github.com/bblfsh/sdk/issues/325